### ISSUE TITLE
Create cheatsheet

### ISSUE THREAD

**Issue by jhollway** [globalgov/qData#99]

**Comment by EstherPeev** [comment]
@jhollway Please find below the first draft of the cheat sheet that I have prepared for qCreate. I am still working on the one for qData.

[https://app.zenhub.com/files/291753904/00b24121-6e84-424c-8e6d-78ffde4047a0/download](https://app.zenhub.com/files/291753904/00b24121-6e84-424c-8e6d-78ffde4047a0/download)

Also @henriquesposito let me know if you have any ideas or suggestions on how to improve it or if you think something is unclear.

**Comment by jhollway** [comment]
This is excellent @EstherPeev ! Really useful examples, etc. I'll take a closer look tomorrow, but one thing that might be nice to include if we have the space is something about where testing enters into the workflow.

**Comment by EstherPeev** [comment]
@jhollway Yes thank you I indeed forgot the test parts, I'll add it to the cheat sheet. 
@henriquesposito Yes, I agreee with you I will see if I can add the little blue boxes for all the functions to explain the arguments.

**Comment by EstherPeev** [comment]
Here is the second draft: 

[https://app.zenhub.com/files/291753904/dbe38a47-ce8e-4c0e-b7bb-e33f181e9c33/download](https://app.zenhub.com/files/291753904/dbe38a47-ce8e-4c0e-b7bb-e33f181e9c33/download)

I have added few words on the tests functionality in the process part and I've also adjusted the blue boxes and the blue square background. Please let me know if anything else is missing or need to be modified.

**Comment by EstherPeev** [comment]
Here is the third draft: 

[https://app.zenhub.com/files/291753904/a2ad3269-c690-47b9-a682-09b05f59454a/download](https://app.zenhub.com/files/291753904/a2ad3269-c690-47b9-a682-09b05f59454a/download)

I tried to incorporate all of your comments/suggestions. I have also added the background color for the bottom part to match with the upper part. Please let me know what you think of this one. 

@henriquesposito @jhollway

**Comment by henriquesposito** [comment]
I like the colors a lot, great job @EstherPeev ! Maybe even try three different colors or shades to the same color, if possible, because we have three different processes (collect, correct and connect). 

I do think we should take the "" from the arguments and the $ and just leave the simple argument without specifying, especially because we are not using any real examples; for example: code_agreements(dataset, title, date). 

Once again, good job! It does make more sense and it is getting better.
